binder/postBuild

```diff
@@ -1,3 +1,2 @@
-Rscript analyses/deep-review-contrib/install-manual-r-dependencies.r
 sed -i 's/R \[conda env:contrib-viz\]/R \[conda env:conda\]/g' analyses/deep-review-contrib/02.contrib-viz.ipynb
```

Comment:
One of the environment updates changed the kernel behavior in Binder.  I get the kernel not found error again.  Downloading the notebook from Binder shows
```
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "R [conda env:root] *",
   "language": "R",
   "name": "conda-root-r"
  },
```

Can you please try updating the display name and name in these `sed` commands?